### DISCUSSION TITLE
New STT plugin - Deepgram

### DISCUSSION THREAD

**Discussion by JarbasAl** [OpenVoiceOS/ovos-buildroot#154] [Show and tell]
https://github.com/OpenVoiceOS/ovos-stt-plugin-deepgram

**Comment by mushie4282** [comment]
Has anyone been able to install this plugin because I've been having issues. 
![image](https://github.com/OpenVoiceOS/ovos-buildroot/assets/106356342/2ece99c3-f7e7-4722-b4c9-e823d921b7d8)

**Reply by goldyfruit** [reply]
There is no release published on PyPi.

```
pip install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-deepgram.git
```
